src/Sylius/Bundle/ApiBundle/spec/Validator/Constraints/EligibleProductVariantToCartValidatorSpec.php

```diff
@@ -76,7 +83,7 @@ function it_adds_violation_if_product_variant_does_not_exist(
 
         $this->validate(
             new AddItemToCart( 'productVariantCode', 1),
```

Comment:
Talking about the amount of paths that go through this service, in case you would like to keep both constraints in one validator you need to duplicate most of the tests here with the change of the constraint and its behavior - ChangeItemQuantityInCart

Plus, can you remove this space before `'` as you already touched this code?